src/main/java/com/glovo/ttglovo/securityManagement/appuser/AppUser.java

```diff
@@ -0,0 +1,172 @@
+package com.glovo.ttglovo.securityManagement.appuser;
+
+import com.glovo.ttglovo.cart.CartItem;
+import lombok.*;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+import java.util.*;
+
+@Builder
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity(name = "AppUser")
+@Table(name = "user_table",
+        uniqueConstraints = {
+                @UniqueConstraint(
+                        columnNames = "email",
+                        name = "uk_email"
+                )
+        }
+)
+public class AppUser implements UserDetails {
+
+    @Id
+    @SequenceGenerator(
+            name = "user_sequence",
+            sequenceName = "user_sequence",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "user_sequence"
+    )
+    @Column(name = "id")
+    private Long id;
```

Comment:
Good idea to use Long for id. Especially where you expect a lot of data.  
Of course for small tables, like dictionary type (with just couple of hundred record tops), it's fine to use Integer.